tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -95,7 +95,7 @@ def test_that_reports_abuse_as_signed_in_user(self, mozwebqa):
 
         Assert.true(home_page.is_the_current_page)
         home_page.header.click_sign_in()
-        home_page.login(user="default")
+        home_page.login(user="new")
```

Comment:
Thanks alot @bobsilverberg and @m8ttyB for the reviews.  I wasn't so pleased with this type of implementing the login. More views on how to implement this are needed. 
So, I simplified the login method, also adding to Base <code>create_new_user</code> and <code>get_default_user</code> methods that should return the correct user.